### PR TITLE
Fix Chat endpoint

### PR DESCRIPTION
**Pull request by bloodbare**
### Description
Describe the proposed changes made in this PR.

### How was this PR tested?
Describe how you tested this PR.

### CODE REVIEW

File: nucliadb/nucliadb/search/api/v1/chat.py [hunk 1]
```diff
@@ -142,42 +138,48 @@ async def generate_answer(
...
-        (
-            relations_results,
-            incomplete_results,
-            queried_nodes,
-            queried_shards,
-        ) = await node_query(kbid, Method.RELATIONS, relation_request, item.shards)
-        yield base64.b64encode(
+
+        if ChatOptions.RELATIONS in features:
+            yield END_OF_STREAM
```
Comment (lferran) [inline nucliadb/nucliadb/search/api/v1/chat.py:154]: should this be nested under the `if ChatOptions.RELATIONS in features:` condition?

File: nucliadb/nucliadb/search/api/v1/chat.py [hunk 2]
```diff
@@ -90,16 +80,22 @@ async def chat_post_knowledgebox(
...
+        req.system = "You help creating new queries based on context"
 
         new_query_elements = []
         async for new_query_data in predict.chat_query(kbid, req):
             new_query_elements.append(new_query_data)
 
         new_query = (b"".join(new_query_elements)).decode()
+        possible_new_query = new_query.split('"')
+        if len(possible_new_query) == 5:
+            new_query = possible_new_query[3]
```
Comment (lferran) [inline nucliadb/nucliadb/search/api/v1/chat.py:98]: Can you add a little comment explaining why this is needed?

Comment (bloodbare) [reply nucliadb/nucliadb/search/api/v1/chat.py:98]: If you do not ask for relations the end of stream is the end of the stream :)

File: nucliadb/nucliadb/search/api/v1/chat.py (hunk 1 above)
Comment (bloodbare) [reply nucliadb/nucliadb/search/api/v1/chat.py:154]: If you do not ask for relations the end of stream is the end of the stream :)

**Pull request merged**